README.rst

```diff
@@ -79,7 +79,7 @@ Package                Minimum Version
 ``boost``                1.46.1
 ``libxml2``              2
 ``libxml++``             2.36
-``python``               2.7 or 3.3+
+``python``               2.7 or 3.10
```

Comment:
Haven't we moved on from Python 2?

Maybe I need to update this for #1588

Reply:
I don't know. I've only ever used cyclus with python 3. If we know it's no longer supported then I'll remove it.